src/ingest_validation_tools/table-schemas/level-2/imc.yaml

```
@@ -39,54 +39,96 @@ fields:
   description: Identifier for the ROI
 - name: acquisition_id
   description: Date and tissue type of acquisition. TODO - On our side, we strongly believe that IDs should not encode semantics. If the two pieces of information are useful, they should be recorded separately.
-- name: after_ablation_image_end_offset
+- name: after_ablation_image_end_offset_value
   description: The number of bytes from the beginning of the MCD file where the respective
     entity is stored as binary
-- name: after_ablation_image_start_offset
+- name: after_ablation_image_end_offset_units
+  description: The units of the offset value.
+  enum: bytes
+- name: after_ablation_image_start_offset_value
   description: The number of bytes from the beginning of the MCD file where the respective
     entity is stored as binary
   type: number
-- name: before_ablation_image_end_offset
+- name: after_ablation_image_start_offset_units
+  description: The units of the offset value.
+  enum: bytes
+- name: before_ablation_image_end_offset_value
   description: The number of bytes from the beginning of the MCD file where the respective
     entity is stored as binary
   type: number
-- name: before_ablation_image_start_offset
+- name: before_ablation_image_end_offset_units
+  description: The units of the offset value.
+  enum: bytes
+- name: before_ablation_image_start_offset_value
   description: The number of bytes from the beginning of the MCD file where the respective
     entity is stored as binary
   type: number
-- name: data_end_offset
+- name: before_ablation_image_start_offset_units
+  description: The units of the offset value.
+  enum: bytes
+- name: data_end_offset_value
   description: The number of bytes from the beginning of the MCD file where the respective
     entity is stored as binary
   type: number
-- name: data_start_offset
+- name: data_end_offset_units
+  description: The units of the offset value.
+  enum: bytes
+- name: data_start_offset_value
   description: The number of bytes from the beginning of the MCD file where the respective
     entity is stored as binary
   type: number
+- name: data_start_offset_units
+  description: The units of the offset value.
+  enum: bytes
 - name: dual_count_start
   description: Threshold for dual counting.
   type: number
 - name: end_datetime
   description: Time stamp indicating end of ablation for ROI
   type: datetime
   format: "%Y-%m-%d %H:%M"
-- name: maxx_um
-  description: Image width of the ROI acquisition in um
+- name: maxx_width_value
+  description: Image width value of the ROI acquisition
   type: number
-- name: maxy_um
-  description: Image height of the ROI acquisition in um
+- name: maxx_width_unit
+  description: Units of image width of the ROI acquisition
+  enum: pixels, um
```

```suggestion
  constraints:
    enum:
      - um
```
@cebriggs7135 : pixels is not a meaningful unit of distance. We went through this several months back; you had found documents saying a pixel was some fraction of an inch, and I explained that that was only true in the context of web design.